locales/fr.yml

```diff
@@ -186,47 +194,48 @@ fr:
     minutes_plural: "minutes"
     email:
     suspended:
-      subject: "Your account has been suspended"
+      subject: "Votre compte a été suspendu"
     banned:
-      subject: "Your account has been banned"
-      body: "In accordance with The Coral Project’s community guidelines, your account has been banned. You are now longer allowed to comment, flag or engage with our community."
+      subject: "Votre compte a été banni"
+      body: "Conformément aux directives communautaires du projet Coral, votre compte a été banni. Vous ne pouvez désormais plus commenter, signaler ou collaborer avec notre communauté."
     confirm:
-      has_been_requested: "A email confirmation has been requested for the following account:"
-      to_confirm: "To confirm the account, please visit the following link:"
-      confirm_email: "Confirm Email"
-      if_you_did_not: "If you did not request this, you can safely ignore this email."
-      subject: "Email Confirmation"
+      has_been_requested: "Une confirmation de l’adresse électronique a été demandée pour le compte suivant :"
+      to_confirm: "Pour confirmer le compte, veuillez suivre le lien suivant :"
+      confirm_email: "Confirmer l’adresse électronique"
+      if_you_did_not: "Si vous n'avez pas fait cette requête, vous pouvez ignorer ce courriel en toute sécurité."
+      subject: "Confirmation adresse électronique"
     password_reset:
-      we_received_a_request: "We received a request to reset your password. If you did not request this change, you can ignore this email."
-      if_you_did: "If you did,"
-      please_click: "please click here to reset password"
+      we_received_a_request: "Nous avons reçu une demande de réinitialisation de votre mot de passe. Si vous n'avez pas demandé cette modification, vous pouvez ignorer ce courriel."
+      if_you_did: "Si vous avez fait cette requête,"
```

Comment:
And here, "Si vous êtes à l’origine de cette requête,"

Reply:
Fixed